R/measures.R

```diff
@@ -2,8 +2,17 @@
 #'
 #'
 #' @param station station ID, see riem_stations()
-#' @param date_start date of start of the desired data, e.g. "2000-01-01"
+#' @param date_start date of start of the desired data, e.g. "2016-01-01"
+#' @inheritParams rlang::args_dots_empty
 #' @param date_end date of end of the desired data, e.g. "2016-04-22"
+#' @param data A vector of strings, representing the data columns to return.
+#' The available options are: all, tmpf, dwpf, relh, drct, sknt, p01i, alti, mslp, vsby, gust, skyc1, skyc2, skyc3, skyc4, skyl1, skyl2, skyl3, skyl4, wxcodes, ice_accretion_1hr, ice_accretion_3hr, ice_accretion_6hr, peak_wind_gust, peak_wind_drct, peak_wind_time, feel, metar, snowdepth # nolint: line_length_linter
+#' @param elev If TRUE, the elevation (m) of the station will be included in the output. # nolint: line_length_linter
+#' @param latlon If TRUE, the latitude and longitude of the station will be included in the output. # nolint: line_length_linter
+#' @param report_type A vector of strings, representing report types to query.
+#' The available options are `"hfmetar"`, `"routine"`, `"specials"`.
+#' Default value is `c("routine", "specials")`.
+#' The report is then in the `metar` column of the output.
```

Comment:
> The report is then in the `metar` column of the output.

FYI I'm removing this comment since it's untrue.  A "report" in this context is a single line of reported weather data.  Each report (line) is one of the 3 available report types.  It's true that IEM modifies the METAR column in the case of `hfmetar`, but no observable changes are made in the METAR column for either `routine` or `specials`.